examples/rn-mbnb/README.md

```diff
@@ -0,0 +1,143 @@
+  * If there is data to be downloaded, this waits for the data to be fully synced before opening the Realm.
+
+This app opens a Realm via `RealmProvider` (see [AuthenticatedApp.tsx](./frontend/app/AuthenticatedApp.tsx)) and passes the configuration as props. We use `OpenImmediately` for new and existing Realm files in order to use the app while offline.
+
+> See [OpenRealmBehaviorConfiguration](https://www.mongodb.com/docs/realm-sdks/js/latest/types/OpenRealmBehaviorConfiguration.html) for possible configurations of new and existing Realm file behaviors.
+
+## Getting Started
+
+### Prerequisites
+
+* [Node.js](https://nodejs.org/)
+* [React Native development environment](https://reactnative.dev/docs/environment-setup?guide=native)
+  * Refer to the **"React Native CLI Quickstart"**.
+
+### Set up an Atlas Database with a Sample Dataset
+
+1. [Deploy a free Atlas cluster](https://www.mongodb.com/docs/atlas/getting-started/#get-started-with-atlas) and create an Atlas database.
+2. [Load the Sample Airbnb Dataset](https://www.mongodb.com/docs/atlas/sample-data/) into your Atlas database.
+    * Several databases and collections exist in the sample dataset, but we will only be using the `sample_airbnb` database and its `listingsAndReviews` collection.
+3. [Create a Search Index](https://www.mongodb.com/docs/atlas/atlas-search/tutorial/create-index/) with an Index Name of `airbnb`.  This will be used for Atlas Search within the application
+
+### Set up an Atlas App Services App via CLI
+
+To import and deploy changes from your local directory to App Services you can use the command line interface:
+
+1. [Set up App Services CLI](https://www.mongodb.com/docs/atlas/app-services/cli/).
+2. In the provided [backend directory](./backend/) (the App Services App), update the following:
+    * Cluster Name
+      * Update the `"clusterName"` in [data_sources/mongodb-atlas/config.json](./backend/data_sources/mongodb-atlas/config.json) to the name of your cluster.
+      * (The default name is `Cluster0`.)
+    * App ID
+      * There is no `"app_id"` defined in [realm_config.json](./backend/realm_config.json) since we will create a brand new App. **If** you for some reason are updating an existing app, add an `"app_id"` field and its value.
+3. [Create and deploy](https://www.mongodb.com/docs/atlas/app-services/cli/appservices-apps-create/) the local directory to App Services:
+```sh
+appservices push --local ./backend
+```
+4. Once pushed, verify that your App shows up in the App Services UI. 
+5. 🥳 You can now go ahead and [install dependencies and run the React Native app](#install-dependencies).
+
+### Install Dependencies
+
+From the project root directory, run:
+
+```sh
+npm install
+```
+
+If developing with iOS, also run:
+
+```sh
+npx pod-install
+```
+
+### Run the App
+
+1. [Copy your Atlas App ID](https://www.mongodb.com/docs/atlas/app-services/reference/find-your-project-or-app-id/#std-label-find-your-app-id) from the App Services UI.
```

Comment:
nit: it also gets populated in the backend/meta.json, at least when the app gets created from the CLI

Reply:
True.  Could also possible pull the AppID directly out of this.